docs/introduction/docker-stream-logging.md

```diff
@@ -0,0 +1,104 @@
+# Logging with docker containers
+If there is need to debug some problem in application that is deployed within many docker containers,
+it is better to have logs from different sources centralized.
+For docker, there is easy way how to do it by sending all logs into streams `STDOUT` and `STDERR`.
+
+## Php-fpm
+In php-fpm docker conainer configuration is set by default that php-fpm service sends error and access log messages into `STDERR` stream.
+`/usr/local/etc/php-fpm.d/docker.conf`
+```
+[global]
+error_log = /proc/self/fd/2
+
+[www]
+; if we send this to /proc/self/fd/1, it never appears
+access.log = /proc/self/fd/2
+
+clear_env = no
+
+; Ensure worker stdout and stderr are sent to the main error log.
+catch_workers_output = yes
+```
+For other docker containers we can easily setup logging of any service inside container like it is for php-fpm.
+
+## Monolog
+Php application uses [symfony/monolog-bundle](https://github.com/symfony/monolog-bundle) for logging. It is easy to setup
+monolog to send logs into streams by modifiying `app/config/config.yml`
+```yaml
+monolog:
+  channels: ["cron", "slow"]
+  handlers:
+    cron:
+      type: stream
+      path: "php://stdout"
+      channels: cron
+      level: debug
+    slow:
+      type: stream
+      path: "php://stdout"
+      channels: slow
+      level: debug
+```
```

Comment:
So we need to change the configuration only for slow and cron logs? What about main monolog log or nested log?